stdlib/public/core/Sequence.swift

```diff
@@ -877,6 +877,13 @@ extension Sequence {
   public func filter(
     _ isIncluded: (Element) throws -> Bool
   ) rethrows -> [Element] {
+    return try _filter(isIncluded)
```

Comment:
That's what they have been doing all along. `filter` is not redefined anywhere near array types. They all inherit the default `Sequence` implementation.